src/components/interactive-builder/edit-question.modal.tsx

```diff
@@ -149,6 +149,8 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
     questionToEdit?.questionOptions.concept &&
     questionToEdit?.questionOptions?.concept !== selectedConcept?.uuid;
   const [addInlineDate, setAddInlineDate] = useState(false);
+  const [toggleLabelTrue, setToggleLabelTrue] = useState('');
+  const [toggleLabelFalse, setToggleLabelFalse] = useState('');
```

Comment:
We should initialize these values to show the existing labels since this is in the edit modal. 
We would have to update the interfaces in `types.ts` to add `toggleOptions` to `questionOptions`.

Reply:
I have updated this together with the issues you had requested in the closed PR.You can look at it